packages/squiggle-lang/src/value/VArray.ts

```diff
@@ -27,7 +27,34 @@ export class VArray
   }
 
   valueToString() {
-    return "[" + this.value.map((v) => v.toString()).join(",") + "]";
+    // For reference, for long types TypeScript returns the following error:
```

Comment:
I think the documentation of this could be improved a fair bit. I asked o1 to do an attempt, a quick look to me makes that seem a fair bit better.

https://chatgpt.com/share/673799fd-d5f4-8003-8cbd-137896284080

Reply:
The functionality of this seems good to me now. Just the documentation / variable naming.